projects/lux/src/lib/input/input.component.html

```diff
@@ -4,7 +4,6 @@
 <input #i1 [luxTooltip]="isGeolocation() ? 'Latitude' : null" placement="top" delay="200"
     [id]="id"
     [type]="domain"
-    [(ngModel)]="value"
```

Comment:
Removing the double databinding breaks the external value double-databinding (see demo in kitchen-sink)